helper/certutil/helpers.go

```diff
@@ -0,0 +1,167 @@
+package certutil
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/sha1"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/mitchellh/mapstructure"
+)
+
+// GetOctalFormatted returns the byte buffer formatted in octal with
+// the specified separator between bytes.
+func GetOctalFormatted(buf []byte, sep string) string {
```

Comment:
All of that code is tested by the tests in the `pki` package, since that package is heavily dependent upon it and the code was split out from it. But if you want separate tests, they could happen.